Jenkinsfile

```diff
@@ -8,8 +8,12 @@
 def images = [
     [image: "docker.io/library/amazonlinux:2",          arches: ["aarch64"]],
     [image: "docker.io/library/centos:7",               arches: ["amd64", "aarch64"]],
-    [image: "quay.io/centos/centos:stream8",            arches: ["amd64", "aarch64"]],
-    [image: "quay.io/centos/centos:stream9",            arches: ["amd64", "aarch64"]],
+    [image: "quay.io/centos/centos:stream8",            arches: ["amd64", "aarch64"]], // EOL: 2024-05-31
+    [image: "quay.io/centos/centos:stream9",            arches: ["amd64", "aarch64"]], // EOL: 2027
+    [image: "docker.io/library/rockylinux:8",           arches: ["amd64", "aarch64"]], // Rocky Linux 8 (EOL: 2029-05-31)
+    [image: "docker.io/library/rockylinux:9",           arches: ["amd64", "aarch64"]], // Rocky Linux 9 (EOL: 2032-05-31)
+    [image: "docker.io/library/almalinux:8",            arches: ["amd64", "aarch64"]], // AlmaLinux 8 (EOL: 2029)
+    [image: "docker.io/library/almalinux:9",            arches: ["amd64", "aarch64"]], // AlmaLinux 9 (EOL: 2032)
     [image: "docker.io/library/debian:buster",          arches: ["amd64", "aarch64", "armhf"]], // Debian 10 (EOL: 2024)
     [image: "docker.io/library/debian:bullseye",        arches: ["amd64", "aarch64", "armhf"]], // Debian 11 (stable)
     [image: "docker.io/library/debian:bookworm",        arches: ["amd64", "aarch64", "armhf"]], // Debian 12 (next stable)
```

Comment:
🙈 Is it ok to update the `//` comments to be aligned with the other ones?

Better to change it now than to do a follow up if that's ok with you 🤗

Reply:
Done